IviDriverPython/1.0/Spec/IviDriverPython.md

```diff
@@ -664,10 +664,10 @@ Documentation = "https://readthedocs.org"
 All IVI-Python driver packages shall include the following files:
 
 - the driver
-- the *README.md* as specified in the [IVI Driver Core Specification](https://github.com/IviFoundation/IviDrivers/blob/main/IviDriverCore/1.0/Spec/IviDriverCore.md)
+- The *README* file. The format of the file shall be markdown (\*.md) or restructured text (\*.rst), where markdown is recommended. The content of the file is specified in the [IVI Driver Core Specification](https://github.com/IviFoundation/IviDrivers/blob/main/IviDriverCore/1.0/Spec/IviDriverCore.md).
```

Comment:
Markdown is a proper noun and should be capitalized. (ref: https://daringfireball.net/projects/markdown/syntax )

reStructuredText is also a proper noun and is styled as "reStructuredText". (ref: https://www.sphinx-doc.org/en/master/usage/restructuredtext/index.html )

```suggestion
- The *README* file. The format of the file shall be Markdown (\*.md) or reStructuredText (\*.rst), where Markdown is recommended. The content of the file is specified in the [IVI Driver Core Specification](https://github.com/IviFoundation/IviDrivers/blob/main/IviDriverCore/1.0/Spec/IviDriverCore.md).
```